applications/rag/README.md

```diff
@@ -1,192 +1,210 @@
+This tutorial walks you through installing the RAG infrastructure in a GCP project, generating vector embeddings for a sample [Kaggle Netflix shows](https://www.kaggle.com/datasets/shivamb/netflix-shows) dataset and prompting the LLM with context.
 
-### (Optional) Use an existing cluster
+# Prerequisites
 
-By default, Terraform creates an Autopilot cluster on your behalf. If you prefer to manage your own cluster, you will need to set `create_cluster = false` in the following section. Creating a long-running cluster is also better for development, allowing you to iterate on Terraform components without recreating the cluster every time.
+### Install tooling (required)
 
-You can use gcloud to create a GKE Autopilot cluster. Note that RAG requires the latest Autopilot features, available on the `RAPID` release channel.
+Install the following on your computer:
+* [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+* [Helm](https://helm.sh/docs/intro/install/)
+* [Gcloud](https://cloud.google.com/sdk/docs/install)
+
+### Bring your own cluster (optional)
+
+By default, this tutorial creates an Autopilot cluster on your behalf. We highly recommend following the default settings.
+
+If you prefer to manage your own cluster, set `create_cluster = false` in the following section under `workloads.tfvars`. Creating a long-running cluster may be better for development, allowing you to iterate on Terraform components without recreating the cluster every time.
+
+Use gcloud to create a GKE Autopilot cluster. Note that RAG requires the latest Autopilot features, available on the `RAPID` release channel.
 
 ```
 CLUSTER_NAME=rag-cluster
-CLUSTER_REGION=us-central1
+CLUSTER_REGION=us-east4
 
 gcloud container clusters create-auto ${CLUSTER_NAME:?} \
   --location ${CLUSTER_REGION:?} \
   --release-channel rapid
 ```
 
-### Enable [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) (Pre-existing network only)
+### Bring your own VPC (optional)
+
+By default, this tutorial creates a new network on your behalf with [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) already enabled. We highly recommend following the default settings.
 
-By default, Terraform uses the pre-existing VPC network (`default`) in your GCP project.
-CloudSQL requires [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) to be enabled in the VPC network.
+If you prefer to use your own VPC, set `create_network = false` in the following section under `workloads.tfvars`. This also requires enabling [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) for your VPC. Without Private Service Connect, the RAG components cannot connect to the vector DB:
```

Comment:
Likewise, let's add a link to the section:

`in the [Installation](#installation) section. This also requires...`

(Likewise, remove workload.tfvars part)